projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1927,11 +1927,7 @@ protected static function build_sprintf( $args ) {
 	 */
 	public function publicize_connections_url( $source = 'calypso-marketing-connections' ) {
 		if ( $this->use_admin_ui_v1() && current_user_can( 'manage_options' ) ) {
-			$is_social_active = defined( 'JETPACK_SOCIAL_PLUGIN_DIR' );
-
-			$page = $is_social_active ? 'jetpack-social' : 'jetpack#/sharing';
```

Comment:
Should we still be doing this but checking the admin-page feature flag too? I'm not sure about versioning of the package etc, but the `jetpack-social` page won't be available in Jetpack until we turn the feature flag on.

Reply:
@pablinos how about this? If the flag is set we go to social, otherwise we do as before